src/neuroconv/datainterfaces/ophys/requirements.txt

```diff
@@ -1 +1 @@
-roiextractors>=0.5.3
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@main
```

Comment:
@CodyCBakerPhD 
I'm leaving this PR in draft until a new version is cut for `roiextractors`, just to keep in mind this is ready